website/astro.config.ts

```
@@ -10,8 +10,9 @@
~import rehypeAstroRelativeMarkdownLinks from 'astro-rehype-relative-markdown-links'
~import astrobook from 'astrobook'
 import { fdir } from 'fdir'
 import rehypeAutolinkHeadings from 'rehype-autolink-headings'
```

ESLint

[error] 'rehypeAutolinkHeadings' is defined but never used. Allowed unused vars must match /^_/u.

(@typescript-eslint/no-unused-vars)